docs/ruleset_functions/get_HVAC_systems_primarily_serving_comp_rooms.md

```diff
@@ -3,33 +3,32 @@
 **Description:** Returns a list of HVAC systems in which greater than 50% of the area served by the HVAC system is computer room space.  
 
 **Inputs:**  
-- **U,B, or P-RMR**: To develop a list of HVAC systems in which greater than 50% of the area served by the HVAC system is computer room space.    
+- **RMD**: To develop a list of HVAC systems in which greater than 50% of the area served by the HVAC system is computer room space.    
 
 **Returns:**  
-- **hvac_systems_primarily_serving_comp_rooms_list_x**: A list of hvac systems in which greater than 50% of the area served by the HVAC system is computer room space.    
+- **hvac_systems_primarily_serving_comp_rooms_list**: A list of hvac systems in which greater than 50% of the area served by the HVAC system is computer room space.    
  
 **Function Call:**  
 
 1. get_hvac_zone_list_w_area()  
 
 ## Logic:   
-- Create RMR object from RMR function input (RMR = function input): `X_RMR = RMR`
-- Get dictionary list of zones and the total floor area served by each HVAC system: `hvac_zone_list_w_area_dict_x = get_hvac_zone_list_w_area (X_RMR)`
-- For each hvac_x in the X_RMR: `for hvac_x in X_RMR...HeatingVentilatingAirConditioningSystem:`
-    - Get the total floor area served by the hvac system: `hvac_sys_total_floor_area_x = hvac_zone_list_w_area_dict_x[hvac_x.id]["TOTAL_AREA"]`
+- Get dictionary list of zones and the total floor area served by each HVAC system: `hvac_zone_list_w_area_dict = get_hvac_zone_list_w_area(RMD)`
+- For each hvac_x in the X_RMD: `for hvac in RMD...HeatingVentilatingAirConditioningSystem:`
```

Comment:
`for hvac in RMD...HeatingVentilatingAirConditioningSystem:`, change `hvac` to `hvac_x`